cleaner.py

```diff
@@ -202,13 +202,13 @@ def clean_command(text: str):
         clean_line = clean_line.replace("[INFO] ", "")
         clean_line = clean_ports_from_IP(clean_line)
 
-        if clean_line.__contains__("logged in with entity id"):
+        if "logged in with entity id" in clean_line:
```

Comment:
This could also use the new `match` syntax added in [Python 3.10](https://peps.python.org/pep-0636/)